src_py/sprite.py

```diff
@@ -1432,6 +1482,11 @@ def __init__(self, ratio):
         """
         self.ratio = ratio
 
+    def __repr__(self):
+        """
+        Turn the class into a string.
+        """
+        return 'ratio: ' + str(self.ratio)
```

Comment:
This doesn't follow a standard repr format. Is repr useful here?

Reply:
I added those quickly because pylint won't let you can't have a function with only one public method. 

It being `__repr__` isn't particularly important, just needs to be something and that seemed like it might be marginally useful. I'll try to make them have the proper format like the other examples in the sprite module.